scripts/prerelease.mjs

```diff
@@ -0,0 +1,75 @@
+import {
+  getNextVersion,
+  getCurrentVersion,
+  gitPush,
+  gitPushTags,
+} from '@coveo/semantic-monorepo-tools';
+import {resolve} from 'path';
+import {execute} from './exec.mjs';
+import {commitVersionBump, stageAll, tagPackages} from './git.mjs';
+import {
+  packageDirsNpmTag,
+  getPackageDefinitionFromPackageDir,
+  updatePackageVersion,
+} from './packages.mjs';
+
+/**
+ * @typedef {import('./packages.mjs').PackageDefinition} PackageDefinition
+ */
+
+export const prereleaseSuffix = 'pre';
+
+/**
+ * @param {string} version
+ */
+export function isPrereleaseVersion(version) {
+  return !!version.match(new RegExp(`-${prereleaseSuffix}\\.[0-9]+\$`));
+}
```

Comment:
- [ ] Use `semver.prerelease`

Reply:
I assumed I couldn't import `semver` because it's a CommonJS module, but I didn't know that this only prevents deconstructed imports. TIL.